xlgui/widgets/playlist_columns.py

```diff
@@ -539,6 +539,39 @@ class CommentColumn(Column):
     autoexpand = True
     # Remove the newlines to fit into the vertical space of rows
     formatter = TrackFormatter('${comment:newlines=strip}')
+    cellproperties = {'editable': True}
+
+    def __init__(self, *args):
+        Column.__init__(self, *args)
+        self.cellrenderer.connect('edited', self.on_edited)
+        self.cellrenderer.connect('editing-started', self.on_editing_started)
+
+    def on_edited(self, cellrenderer, path, new_text):
+        # Undo newline escaping
+        new_text = new_text.decode('unicode-escape')
+
+        # Set comment
+        model = self.get_tree_view().get_model()
+        iter = model.get_iter(path)
+        track = model.get_value(iter, 0)
+
+        track.set_tag_raw("comment", new_text)
+        event.log_event('comment_changed', self, new_text)
```

Comment:
Why is this extra event added? When tags are changed an event already is emitted, no need to add a special event for this.

... sorry for not using the review feature and doing a bunch of comments instead.

Reply:
Ah, I probably added that event logging for debug purposes while developing the patchset (although as you said, it is redundant), and forgot to remove it. Will push a revised patchset without it.